src/gui/AdvancedBSAModule.cpp

```diff
@@ -46,6 +46,7 @@ void AdvancedBSAModule::settings_to_ui(const Settings &settings)
     ui_->dontMakeLoaded->setChecked(!profile.bsa_make_dummy_plugins);
     ui_->dontRemoveFiles->setChecked(!profile.bsa_remove_files);
     ui_->dontCompress->setChecked(!profile.bsa_allow_compression);
+    ui_->makeOverrides->setChecked(profile.bsa_make_overrides);
```

Comment:
We should only show this button when the game supports override, don't you think?

Reply:
Yep, that definitely makes sense.